package.json

```diff
@@ -11,6 +11,7 @@
     "start:msw": "API_MODE=msw vite",
     "start:nexus": "API_MODE=nexus vite",
     "start:dogfood": "API_MODE=dogfood vite",
+    "preview": "API_MODE=msw npm run build && cp mockServiceWorker.js dist/ && vite preview",
```

Comment:
Added this because I added more content to an existing patch in a commit, which made Vercel break. Turns out it caches your `node_modules` at the end of the run, and `patch-package` only knows not to apply a patch if it's entirely applied.

https://github.com/ds300/patch-package/issues/37 has some related chatter.